<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diag_lipsync_vsqx</class>
 <widget class="QDialog" name="diag_lipsync_vsqx">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>283</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>365</width>
    <height>283</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>365</width>
    <height>283</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import VSQX</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBox_transition_duration">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_performer">
     <property name="text">
      <string>Performer:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="checkBox_delete_existing">
     <property name="text">
      <string>Delete existing</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_exclude_premeasure">
     <property name="text">
      <string>Exclude pre-measure</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_transition_timeout">
     <property name="text">
      <string>Transition timeout:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_offset">
     <property name="text">
      <string>Offset (msec):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_transition_duration">
     <property name="text">
      <string>Transition duration:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_format">
     <property name="text">
      <string>Format:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBox_track">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBox_performer">
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBox_transition_timeout">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>50000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_track">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Track (0=all):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBox_offset">
     <property name="minimum">
      <number>-100000</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_format">
     <item>
      <property name="text">
       <string>PDA 2.00/Future Tone f mots</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="checkBox_splitDiphthongs">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Split diphthongs</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>diag_lipsync_vsqx</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>diag_lipsync_vsqx</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
